pkg/cloud/gcp/vm-instance-status.go

```diff
@@ -40,43 +24,57 @@ func GetVMInstanceStatus(instanceName string, gcpProjectID string, instanceZone
 }
 
 //InstanceStatusCheckByName is used to check the status of all the VM instances under chaos
-func InstanceStatusCheckByName(autoScalingGroup string, delay, timeout int, check string, instanceNames string, gcpProjectId string, instanceZones string) error {
+func InstanceStatusCheckByName(computeService *compute.Service, managedInstanceGroup string, delay, timeout int, check string, instanceNames string, gcpProjectId string, instanceZones string) error {
+
 	instanceNamesList := strings.Split(instanceNames, ",")
+
 	instanceZonesList := strings.Split(instanceZones, ",")
-	if autoScalingGroup != "enable" && autoScalingGroup != "disable" {
-		return errors.Errorf("Invalid value for AUTO_SCALING_GROUP: %v", autoScalingGroup)
+
+	if managedInstanceGroup != "enable" && managedInstanceGroup != "disable" {
+		return errors.Errorf("invalid value for MANAGED_INSTANCE_GROUP: %v", managedInstanceGroup)
 	}
+
 	if len(instanceNamesList) == 0 {
-		return errors.Errorf("No instance name found to stop")
+		return errors.Errorf("no instance name found to stop")
 	}
+
 	if len(instanceNamesList) != len(instanceZonesList) {
-		return errors.Errorf("The number of instance names and the number of regions is not equal")
+		return errors.Errorf("the number of instance names and the number of regions is not equal")
 	}
-	log.Infof("[Info]: The instances under chaos(IUC) are: %v", instanceNamesList)
+
+	log.Infof("[Info]: The instances under chaos (IUC) are: %v", instanceNamesList)
+
 	if check == "pre-chaos" {
-		return InstanceStatusCheckPreChaos(instanceNamesList, gcpProjectId, instanceZonesList)
+		return InstanceStatusCheckPreChaos(computeService, instanceNamesList, gcpProjectId, instanceZonesList)
 	}
-	return InstanceStatusCheckPostChaos(timeout, delay, instanceNamesList, gcpProjectId, instanceZonesList)
+
+	return InstanceStatusCheckPostChaos(computeService, timeout, delay, instanceNamesList, gcpProjectId, instanceZonesList)
 }
 
 //InstanceStatusCheckPreChaos is used to check whether all VM instances under chaos are running or not without any re-check
-func InstanceStatusCheckPreChaos(instanceNamesList []string, gcpProjectId string, instanceZonesList []string) error {
+func InstanceStatusCheckPreChaos(computeService *compute.Service, instanceNamesList []string, gcpProjectId string, instanceZonesList []string) error {
+
 	for i := range instanceNamesList {
-		instanceState, err := GetVMInstanceStatus(instanceNamesList[i], gcpProjectId, instanceZonesList[i])
+
+		instanceState, err := GetVMInstanceStatus(computeService, instanceNamesList[i], gcpProjectId, instanceZonesList[i])
 		if err != nil {
 			return err
 		}
+
 		if instanceState != "RUNNING" {
-			return errors.Errorf("failed to get the vm instance '%v' in running sate, current state: %v", instanceNamesList[i], instanceState)
+			return errors.Errorf("%s vm instance is not in RUNNING state, current state: %v", instanceNamesList[i], instanceState)
```

Comment:
```suggestion
			return errors.Errorf("%s instance is not in RUNNING state, current state: %v", instanceNamesList[i], instanceState)
```
At some places `instance` is used and some `vm instance` is used, we can use a single representation (can improve this in other experiments as well)